submissions/best-of-its-genre-nanogamer7/origins.md

```diff
@@ -0,0 +1,46 @@
+## Best of its genre - Nanogamer7
+
+We've all heard of Factorios insanely [good ratings](https://steamdb.info/stats/gameratings/) - consistently rising to second place after Portal 2 again and again.[^1] Obviously many others will try to imitate this success in the automation genre, but where did Factorio get inspired from? That's what I'll be trying to answer in this first article of hopefully a multi-part series.
+
+### Automation defined
+
+The first difficult question we'll have to answer though, would be what automation even is. In Cities:Skylines for example you are building your city, which then (theoretically) functions autonomously. The problem here is, your job isn't to automate - the cars drive themselves, once you build roads - but to manage it. In Minecraft on the other hand, you are the one to make the automation even possible, just with pistons and hoppers, instead of inserters and belts. Comparing these two examples, I think you can loosely define the automation genre as *"making stuff that makes stuff"*.
+
+However that definition is still a bit vague: From what I know about e.g. Dwarf Fortress, you could consider enabling dwarfs to *make stuff* as falling under that definition. But let's not get to far from Factorio here, maybe someone else will explore this further in the future. *\*wink wink\**
```

Comment:
Its sounds like youre trying to hint that DF doesnt have automation, but it does according to your definition of automation. 

And I like this reference :P I just don't think I will do it within the next month, so I don't know if we should add it. Not sure though